cluster/gce/gci/mounter/mounter

```diff
@@ -23,14 +23,25 @@ MOUNTER_USER=root
 RKT_BINARY=/home/kubernetes/bin/rkt
 
 function gc {
-    ${RKT_BINARY} gc --grace-period=0s &> /dev/null
+    # Attempt to garbage collect rkt pods with 5 retries.
+    # Rkt pods end up creating new copies of mounts on the host. Hence it is ideal to clean them up right away.
+    attempt=0
+    until [ $attempt -ge 5 ]; do
+	${RKT_BINARY} gc --grace-period=0s && break
+	attempt=$[$attempt+1]
+	sleep 1
+    done
```

Comment:
It should only garbage collect `completed` pods. I'd expect `rkt` to synchronize across parallel GC operations. This is an issue to consider even without this retry.